web/html/src/manager/recurring/recurring-actions-edit.tsx

```diff
@@ -184,6 +193,14 @@ class RecurringActionsEdit extends React.Component<Props, State> {
     return Promise.resolve(states);
   };
 
+  onSelectPlaybook = (playbook) => {
+    let { details } = this.state;
+    details.playbookPath = playbook?.playbookPath;
+    details.inventoryPath = playbook?.inventoryPath;
+    details.flushCache = playbook?.flushCache;
+    this.setState({ details });
```

Comment:
I think something along these lines would be more idiomatic React, since then we don't mutate what we have in the state directly, even if we `setState` after. But please check if this matches your expected behavior, I don't know if I'm missing some context here.

```suggestion
    this.setState({
      details: {
        ...this.state.details,
        playbookPath: playbook?.playbookPath,
        inventoryPath: playbook?.inventoryPath,
        flushCache: playbook?.flushCache,
      }
    });
```